centralstore/smartwrapper_test.go

```diff
@@ -433,14 +434,14 @@ func BenchmarkStoreGetTracesForState(b *testing.B) {
 	}
 }
 
-func cleanupRedisStore(store BasicStorer) error {
+func cleanupRedisStore(t testing.TB, store BasicStorer) error {
```

Comment:
Given that you're logging and not returning an error, should probably remove the return parameter.